python/simulation.py

```diff
@@ -2107,3 +2107,11 @@ def interpolate(n, nums):
             res.extend(np.linspace(low, high, n + 1, endpoint=False).tolist())
 
     return res + [nums[-1]]
+
+
+def meep_vec(it):
```

Comment:
the name `meep_vec` kind of rubs me the wrong way, since the `meep_` prefix is redundant with the module name.

My ideal interface would be something that simply dispatched on the argument type, e.g `mp.vec(1,2,3)` and `mp.vec([1,2,3])` would both work.

Reply:
Updated to your ideal interface.